packages/vm/core/accounts/impl.go

```diff
@@ -170,12 +170,6 @@ func transferAccountToChain(ctx isc.Sandbox, optionalGasReserve *uint64) {
 		panic(vm.ErrUnauthorized)
 	}
 
-	// if the caller contract is on the same chain the transfer would end up
-	// in the same L2 account it is taken from, so we do nothing in that case
-	if callerContract.ChainID().Equals(ctx.ChainID()) {
-		return
-	}
-
```

Comment:
`callerContract.ChainID().Equals(ctx.ChainID())` is true for single chain node :thinking:

Reply:
I think we should remove the whole entry point transferAccountToChain, since it uses cross-chain requests